app-commons-test/src/test/scala/org/bitcoins/commons/dlc/OutcomeValueFunctionTest.scala

```diff
@@ -0,0 +1,306 @@
+package org.bitcoins.commons.dlc
+
+import org.bitcoins.commons.jsonmodels.dlc._
+import org.bitcoins.core.currency.Satoshis
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+import org.scalacheck.Gen
+
+import scala.math.BigDecimal.RoundingMode
+
+class OutcomeValueFunctionTest extends BitcoinSUnitTest {
```

Comment:
yeah, I find these quite annoying because scala doesn't have type-level functions and also doesn't provide public access to types like Vector2, Vector3 etc.

These are not actual issues as the input to `nPoints` determines the length of the output (and this is test code anyway, the tests would just fail). Not really sure what to do, if we really cared I could add a `case _ => fail` to every single test :/

Reply:
eh, guess I'll just do that, not as many tests as I thought use `nPoints`